src/routes/timeline/timelineData.ts

```diff
@@ -95,110 +102,113 @@ export const timelineItems: TimelineItem[] = [
 		icon: '🍾',
 		status: 'milestone',
 		title: 'First deployment of MVP',
-		summary:
-			'On September 7 Felt was deployed to a persistant staging environment on the cloud for the first time.',
+		summary: 'On September 7 Felt was deployed to the cloud for the first time.',
 		date: 'Q3 2021',
 	},
 	{
 		icon: '↗️',
 		status: 'completed',
 		title: 'Published the "Consentful Onboarding" sketch',
 		summary:
-			'We published a sketch comparing consentful & non-consentful onboarding experinces side by side <a href="https://ui.felt.dev/sketch/onboard">Consentful Onboarding</a>',
+			'We published a <a href="https://ui.felt.dev/sketch/onboard">Consentful Onboarding Sketch</a> comparing hypothetical experiences with help from <a href="https://imjane.net/">Jane Im</a> (<a href="https://github.com/trusttri">@trusttri</a>) and her work on <a href="https://consentful.systems/">consentful.systems</a>.',
 		date: 'Q3 2021',
 	},
 	{
 		icon: '🎭',
 		status: 'completed',
 		title: 'Actors v1',
-		summary: 'The multi-Actor system (originally called Personas) is put into place.',
+		summary: 'Added the multi-Actor system, originally "personas".',
 		date: 'Q3 2021',
 	},
 	{
 		icon: '🏡',
 		status: 'completed',
 		title: 'Spaces v1',
-		summary: 'The Space data model & first View, Chat, is implemented.',
+		summary: 'Added the Space data model and Chat, the first View.',
 		date: 'Q2 2021',
 	},
 	{
 		icon: '🌏',
 		status: 'completed',
 		title: 'Hubs v1',
-		summary: 'The core data model for Hubs (initially Communities) is implemented.',
+		summary: 'Added the data model for Hubs, originally "communities".',
 		date: 'Q2 2021',
 	},
 	{
 		icon: '🔐',
 		status: 'completed',
 		title: 'Accounts v1',
-		summary: 'The initial core Account system is implemented.',
+		summary: 'Added the Account system so we can sign up and in and out.',
 		date: 'Q2 2021',
 	},
 	{
 		icon: '🌱',
 		status: 'milestone',
-		title: 'Felt Server officially spins up!',
-		summary: 'On April 5th Hamilton & Ryan begin development on the core Felt repo full time!',
+		title: 'felt-server boots up',
+		summary:
+			'On April 5th Hamilton and Ryan begin development on <a href="https://github.com/feltjs/felt-server">the main Felt repo</a> full-time!',
 		date: 'Q2 2021',
 	},
 	{
 		icon: '⌨️',
 		status: 'completed',
-		title: 'Start `felt-template` & `felt` component library',
-		summary: 'The first repositories in the Felt toolkit are spun up.',
+		title:
+			'Started <a href="https://github.com/felt/felt-template">felt-template</a> and <a href="https://github.com/felt/felt-ui">felt-ui</a>',
+		summary: 'Created the first <a href="https://github.com/feltjs">@feltjs</a> repos.',
 		date: 'Q1 2021',
 	},
 	{
 		icon: '🧠',
 		status: 'completed',
-		title: 'Pre-Production',
-		summary: 'The Felt team goes headsdown on preproduction, brainstorming, and technical prep.',
+		title: 'Preproduction',
```

Comment:
```suggestion
		title: 'Pre-production',
```
*pedantically* Pre-production is a hyphenated word 🤓